deploy/crds/keda.sh_scaledjobs_crd.yaml

```diff
@@ -1416,7 +1416,6 @@ spec:
                                       type: string
                                   required:
                                   - containerPort
-                                  - protocol
```

Comment:
Please see this issue: https://github.com/kedacore/keda/issues/927 there's currently bug in the libraries for generating CRDs, we need to keep the `protocol` property in here. For now it is needed to manually add them on those places.

Reply:
this is the only thing that's blocking the merge of this PR from my POV

Reply:
Done